dsl/api.go

```diff
@@ -63,29 +62,43 @@ func API(name string, fn func()) *design.APIExpr {
 //    })
 //
 func Title(val string) {
-	if a, ok := eval.Current().(*design.APIExpr); ok {
-		a.Title = val
+	if s, ok := eval.Current().(*expr.APIExpr); ok {
+		s.Title = val
 		return
 	}
 	eval.IncompatibleDSL()
 }
 
-// Version sets the API version. It is used by the generated OpenAPI
-// specification.
-//
-// Version must appear in a API expression.
-//
-// Version accepts a single string argument.
-//
-// Example:
-//
-//    var _ = API("divider", func() {
-//        Version("1.0")
-//    })
-//
+// Version specifies the API version. One design describes one version.
 func Version(ver string) {
-	if a, ok := eval.Current().(*design.APIExpr); ok {
-		a.Version = ver
+	if s, ok := eval.Current().(*expr.APIExpr); ok {
+		s.Version = ver
+		return
+	}
+	eval.IncompatibleDSL()
+}
+
+// Contact sets the API contact information.
```

Comment:
Should we add header documentation for these?

Reply:
Fixed